packages/react-native/src/Authenticator/Authenticator.tsx

```diff
@@ -10,8 +10,7 @@ import {
   useAuthenticatorInitMachine,
 } from '@aws-amplify/ui-react-core';
 
-import { configureComponent } from '@aws-amplify/ui';
-
+import { useSetUserAgent } from '@aws-amplify/ui-react/internal';
```

Comment:
`@aws-amplify/ui-react` is not a dependency of `@aws-amplify/ui-react-native` nor should it be